GitHub-Test-DHprojects/GengC_Test4.md

```diff
@@ -0,0 +1,3 @@
+# DH projects review
+This project analyzes the world's most popular K-Pop idols and groups. They explore questions like "What is K-Pop?" on their primary about page. They also talk about lots of different groups. They included background information, members, fandom names, etc., on each page. They had self-portraits of the group members to the project, which I thought was an excellent idea, that people could see who was in each group and get a better feel. On top of the photographs, they included charts comparing the roles of members of the groups and graphs corresponding to the number of choruses in various songs. They had photos of their code on the Code page, and some Korean words looked very cool. They did a great job, that if we could have more different types of music groups, that would be better. 
```

Comment:
Hey @gzc5211 I really like how in depth you went with this it was well thought out and interesting keep up the fantastic work!